build/README.md

```diff
@@ -0,0 +1,15 @@
+Mimiker Build System
+======================
+
+### Command line options
+
+The following command line options are provided:
+- BOARD: The target board. Defaults to `malta`.
```

Comment:
I think it would be nice to list here other boards

Reply:
Done.